src/sentry/tasks/deliver_from_outbox.py

```diff
@@ -3,12 +3,12 @@
 import math
 from typing import Any, Type
 
+import sentry_sdk
```

Comment:
Curious novice question: Is there a preference for `import sentry_sdk` over `from sentry_sdk.api import capture_exception`?
They both bring in the same `capture_exception` method.

Reply:
I'm not sure really. Both `sentry_sdk.api.capture_exception` and `sentry_sdk.capture_exception`. I needed to use scope modification as well and it felt simpler to import the entire module :shrug: